### DISCUSSION TITLE
How do I set the response language?

### DISCUSSION THREAD

**Discussion by MatsGej** [dahlia/wikidata#52] [Q&A] [answered]

**Comment by dahlia** [comment]
Could you elaborate what you exactly want?  What do you mean by “response language”?

**Comment by MatsGej** [comment]
Hi
I wanted to be able to control the language of the text returned from wikidata. But I could not understand how to do that? Only got it in English!

Med vänlig hälsning, Best regards
Mats
Sent from my mobile

30 okt. 2022 kl. 14:06 skrev Hong Minhee (洪 民憙) ***@***.***>:

﻿

Could you elaborate what you exactly want? What do you mean by “response language”?

**Comment by dahlia** [comment, marked as answer]
Entity objects contain all languages besides English.  Let me elaborate with a demo session:

```pycon
>>> from wikidata.client import Client
>>> client = Client()
>>> earth = client.get('Q2')
>>> earth.label
m'Earth'
>>> earth.description
m'third planet from the Sun in the Solar System'
```

Note that these strings have prefix `m`.  The prefix means these are multilingual strings.  If you look at the type of them you could notice these are not `str`:

```pycon
>>> type(earth.label)
<class 'wikidata.multilingual.MultilingualText'>
>>> type(earth.description)
<class 'wikidata.multilingual.MultilingualText'>
```

Actually, `MultilingualText` is more similar to dictionaries than strings, because there mappings of language tags to corresponding texts:

```pycon
>>> earth.label['en']
'Earth'
>>> earth.label['fr']
'Terre'
>>> earth.label['zh']
'地球'
>>> earth.description['en']
'third planet from the Sun in the Solar System'
>>> earth.description['fr']
'troisième planète à partir du Soleil dans le système solaire'
>>> earth.description['zh']
'太陽系中距離太陽第三近的行星'
```

**Comment by MatsGej** [comment]
Thanks, that was easy!!!
